symfony/plugins/orangehrmBuzzPlugin/lib/dao/BuzzNotificationDao.php

```diff
@@ -0,0 +1,216 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
```

Comment:
header is wrong

Reply:
Oops. But seems fine. Can you check this [https://github.com/orangehrm/orangehrm/blob/master/symfony/plugins/orangehrmPimPlugin/modules/pim/actions/addEmployeeAction.class.php](https://github.com/orangehrm/orangehrm/blob/master/symfony/plugins/orangehrmPimPlugin/modules/pim/actions/addEmployeeAction.class.php)